src/test/java/org/folio/rest/impl/ModTenantAPIIntegrationTest.java

```diff
@@ -0,0 +1,98 @@
+package org.folio.rest.impl;
+
+import static io.restassured.RestAssured.*;
+import static org.folio.rest.impl.OkapiMockServer.OAI_TEST_TENANT;
+import static org.mockserver.model.HttpRequest.request;
+import static org.mockserver.model.HttpResponse.response;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.http.ContentType;
+import java.nio.file.Path;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.mockserver.client.MockServerClient;
+import org.mockserver.model.MediaType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.MockServerContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.images.builder.ImageFromDockerfile;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+@Disabled
+@Testcontainers
+class ModTenantAPIIntegrationTest {
```

Comment:
The class name should be `ModTenantAPIIT` following the maven-surefire-plugin naming scheme:
https://maven.apache.org/surefire/maven-failsafe-plugin/examples/inclusion-exclusion.html
This test uses the Dockerfile that requires mod-oai-pmh-fat.jar. The jar is not available when maven-failsave-plugin runs. It is available when maven-surefire-plugin runs.

Reply:
Thank you, Julian.